gazebo/private/gz_configure_file.bzl

```diff
@@ -91,4 +93,5 @@ def gz_configure_file(
         name = name,
         hdrs = [out],
         includes = ["include"],
+        deps = deps,
```

Comment:
I think, in practice, we are not using any `**kwargs` going into the `_gz_configure_file` here, do you want to just use it with `cc_library` so that we can pass whatever else we may need?